src/v2/view-builder/internals/BaseForm.js

```diff
@@ -0,0 +1,37 @@
+import { Form, loc } from 'okta' ;
+import FormInputFactory from './FormInputFactory';
+
+export default Form.extend({
+
+  layout: 'o-form-theme',
+  className: 'ion-form',
+  hasSavingState: true,
+  autoSave: false,
+  noCancelButton: true,
+  title: '',
```

Comment:
Should we provide a default title in the BaseForm ?

Reply:
Good point. Any suggestion?